app/services/testdata/GeneratorConfig.scala

```diff
@@ -17,9 +17,10 @@
 package services.testdata
 
 import model.EvaluationResults.Result
+import org.joda.time.LocalDate
```

Comment:
Yep, we want to stop using it but we can't while we're still on Play 2.3 - at least not easily. There's an internal HMRC spike happening to upgrade our libraries etc to play 2.5 - so eventually it will be possible.